stdlib/experimental/count/count.flux

```diff
@@ -0,0 +1,13 @@
+package count
+
+import "experimental"
+
+distinctByTag = (tables=<-, measurement, tag) =>
+	tables
+		|> filter(fn: (r) => r._measurement == measurement)
+		|> keep(columns: [tag])
+  		|> limit(n: 1)
+  		|> group()
+  		|> count(column: tag)
+		|> yield()
```

Comment:
out of curiosity, why is the yield needed here?

Reply:
the error message told me so, I am not sure why either

Reply:
hmm, hopefully @nathanielc will know 🤞